style.css

```
@@ -0,0 +1,51 @@
+* {
+    margin: 0;
+}
+
+.container {
+    margin-top: 100px;
```

Може краще [margin-block-start](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start)?
Він краще працює з локалізацією.
```suggestion
    margin-block-start: 100px;
```